llarp/dns/message.cpp

```diff
@@ -120,36 +121,32 @@ namespace llarp
         }
         llarp::LogDebug(qd);
       }
-
       for(auto& an : answers)
       {
-        if(!an.Decode(buf))
+        if(not an.Decode(buf))
         {
           llarp::LogError("failed to decode answer");
           return false;
         }
-        llarp::LogDebug(an);
       }
-
-      for(auto& ns : authorities)
+      /*
+      for(auto& auth : authorities)
       {
-        if(!ns.Decode(buf))
+        if(!auth.Decode(buf))
         {
-          llarp::LogError("failed to decode authority");
+          llarp::LogError("failed to decode auth");
           return false;
         }
-        llarp::LogDebug(ns);
       }
-
-      for(auto& ar : additional)
+      for(auto& rr : additional)
       {
-        if(!ar.Decode(buf))
+        if(!rr.Decode(buf))
         {
-          llarp::LogError("failed to decode additonal");
+          llarp::LogError("failed to decode additional");
           return false;
         }
-        llarp::LogDebug(ar);
       }
+      */
```

Comment:
I feel like this commented-out chunk of code is going to bit rot until we forget why it's commented out.

Reply:
all lost, like tears in rain.